scripts/generate-select-validator-report.bash

```diff
@@ -110,56 +126,68 @@ while IFS='|' read -r vote_account apy total_rewards institutional_effective ins
-    echo "---------------------------------------------+-----------------+--------------+-----------+-----------------+--------------------+--------------+---------------+--------------------+"
+    echo "Select Vote Account ($select_count)                     | Name            |       APY | Infl./MEV | Total Rewards | Total Effective | Validator Rewards | Select Effective |  PSR Penalty | Staker Payout | Distributor Payout"
+            DIVIDER="---------------------------------------------+-----------------+-----------+-----------+---------------+-----------------+-------------------+------------------+--------------+---------------+--------------------+"
+            PRINTER_FORMAT="%-44s | %-15s | %9s | %9s | %13s | %15s | %17s | %16s | %12s | %13s | %18s"
+    echo "$DIVIDER"
 
     total_total_rewards=0
     total_effective=0
+    total_validator_rewards=0
+    total_institutional_effective=0
     total_psr_penalty=0
     total_staker_payout=0
     total_distributor_payout=0
 
     for validator_data in "${select_validators[@]}"; do
-        IFS='|' read -r vote_account name apy total_rewards institutional_effective psr_penalty staker_payout distributor_payout commission_display <<< "$validator_data"
+        IFS='|' read -r vote_account name apy total_rewards total_effective validator_rewards institutional_effective psr_penalty staker_payout distributor_payout commission_display <<< "$validator_data"
 
         formatted_apy=$(format_percentage "$apy")
-        formatted_rewards=$(format_sol_amount "$total_rewards")
-        formatted_effective=$(format_sol_amount "$institutional_effective")
+        formatted_total_rewards=$(format_sol_amount "$total_rewards")
+        formatted_total_effective=$(format_sol_amount "$total_effective")
+        formatted_validator_rewards=$(format_sol_amount "$validator_rewards")
+        formatted_institutional_effective=$(format_sol_amount "$institutional_effective")
         formatted_penalty=$(format_sol_amount "$psr_penalty")
         formatted_staker_payout=$(format_sol_amount "$staker_payout")
         formatted_distributor_payout=$(format_sol_amount "$distributor_payout")
 
-        printf "%-44s | %-15s | %12s | %9s | %15s | %18s | %12s | %13s | %18s\n" \
+        printf "${PRINTER_FORMAT} |\n" \
             "$vote_account" \
             "$name" \
             "$formatted_apy" \
             "$commission_display" \
-            "$formatted_rewards" \
-            "$formatted_effective" \
+            "$formatted_total_rewards" \
+            "$formatted_total_effective" \
+            "$formatted_validator_rewards" \
+            "$formatted_institutional_effective" \
             "$formatted_penalty" \
             "$formatted_staker_payout" \
             "$formatted_distributor_payout"
 
         total_total_rewards=$((total_total_rewards + total_rewards))
-        total_effective=$((total_effective + institutional_effective))
+        total_effective=$((total_effective + total_effective))
```

Comment:
I think this "total variable" will need to be renamed to something like `total_total_effective`, because now the local variable is named the same way as the total one and some unexpected results would be calculated afaik.